1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,31 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let content = document.getElementById("content");
+  let pElement=document.createElement("p");
+  let selectElemet= document.createElement("select");
+  for (let color of colours){
+  let optionElement=document.createElement("option");
+  optionElement.innerText= color
+  optionElement.value= color 
+  selectElemet.appendChild(optionElement)
+  }
+  paragraph.split(" ").forEach(word => {
+    let span= document.createElement("span")
+    span.innerText = word + " "
+  span.addEventListener("click",function(){
+    let selectValue = selectElemet.value
+    if (selectValue ==="yellow"){
+      span.style.backgroundColor= "yellow"
+    } else if( selectValue === "green"){
+      span.style.backgroundColor = "green"
+    } else if(  selectValue === "blue"){
+      span.style.backgroundColor = "blue"
+    } else {span.style.backgroundColor = "white"}
```

Comment:
This works well. Is there an easier way though?  

You could break the handler out to a separate function.  Something like

`span.addEventListener("click", highlightWord);`

I don't think you need to do all the if and else if statements.  You can directly set the background colour from the currently selected colour. So

```
if( something == "green"){
    backgroundColour = "green"
}
```

is the same as `backgroundColour = something`. Can you see?